App.tsx

```diff
@@ -31,7 +30,7 @@ export default function App() {
 const styles = StyleSheet.create({
     wrapper: {
         flex: 1,
-        backgroundColor: Colors.WHITE,
+        backgroundColor: 'white',
```

Comment:
You shouldn't use color literals. They've been moved to constants especially for reusing it and not hardcoding.
```suggestion
        backgroundColor: Colors.WHITE,
```